torch/lib/TH/generic/THTensor.c

```diff
@@ -922,7 +922,7 @@ THDescBuff THTensor_(desc)(const THTensor *tensor) {
   int i;
   for(i = 0; i < tensor->nDimension; i++) {
     if(n >= L) break;
-    n += snprintf(str+n, L-n, "%ld", tensor->size[i]);
+    n += snprintf(str+n, L-n, "%lld", tensor->size[i]);
```

Comment:
nit: is this the right format? I'm looking at https://stackoverflow.com/questions/9225567/how-to-print-a-int64-t-type-in-c and I'm not sure it's exactly what you are looking for. (This appears elsewhere in the codebase as well). Also, there are warning in the CI due to those changes, with warnings like
```
warning: format ‘%lld’ expects argument of type ‘long long int *’, but argument 3 has type ‘int64_t * {aka long int *}’ [-Wformat=]
```